pkg/config/validation/validation.go

```diff
@@ -679,11 +678,15 @@ var ValidateDestinationRule = registerValidateFunc("ValidateDestinationRule",
 			v = appendValidation(v, validateSubset(subset))
 		}
 
-		v = appendValidation(v, validateExportTo(cfg.Namespace, rule.ExportTo, false))
+		v = appendValidation(v,
+			validateExportTo(cfg.Namespace, rule.ExportTo, false, rule.GetWorkloadSelector() != nil))
+
+		v = appendValidation(v, validateWorkloadSelector(rule.GetWorkloadSelector()))
```

Comment:
can we also validate there is a hostname set with workload selector?

Reply:
Added